ccd-definition/ComplexTypes/JudgeAndLegalAdvisor.json

```diff
@@ -0,0 +1,38 @@
+[
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "JudgeAndLegalAdvisor",
+    "ListElementCode": "judgeTitle",
+    "FieldType": "FixedRadioList",
+    "FieldTypeParameter": "JudgeOrMagistrateTitle",
+    "ElementLabel": "Judge or magistrate's title",
+    "SecurityClassification": "Public"
+  },
+  {
+    "LiveFrom": "01/01/2017",
+    "ID": "JudgeAndLegalAdvisor",
+    "ListElementCode": "judgeLastName",
+    "FieldType": "Text",
+    "ElementLabel": "Last name",
+    "FieldShowCondition": "judgeTitle=\"HER_HONOUR_JUDGE\" OR judgeTitle=\"HIS_HONOUR_JUDGE\" OR judgeTitle=\"DEPUTY_DISTRICT_JUDGE\"",
```

Comment:
Couldn't this use `CONTAINS` as they all contain `JUDGE`

Reply:
I flagged this on the PR that implemented this change... it was apparently not possible 🤷‍♂

Reply:
I shall look into